packages/react/src/components/Cta/__stories__/Cta.stories.js

```diff
@@ -12,70 +12,109 @@ storiesOf('Components|CTA', module)
-    // }
-    const data = {
-      text: {
-        copy: 'Nunc Pulvinar Nisi',
-        href: 'https://www.ibm.com',
-        type: type[1],
-      },
-      card: {
-        href: 'https://www.ibm.com',
-        title: 'Nunc Pulvinar Nisi',
-        type: type[1],
-      },
-      feature: {
-        heading: 'Nunc Pulvinar Nisi',
-        card: {
-          href: 'https://www.ibm.com',
-          title: 'Nunc Pulvinar Nisi',
-          type: type[1],
-          image: {
-            defaultImage: 'https://picsum.photos/id/2/672/672',
-            alt: 'featured link image',
-          },
-        },
-      },
-      button: [
-        {
-          href: 'https://www.ibm.com',
-          copy: 'Nunc Pulvinar Nisi',
-          type: type[1],
-        },
-        {
-          href: 'https://www.example.com',
-          copy: 'Nunc Pulvinar Nisi',
-          type: type[1],
-        },
-      ],
-    };
-    const styles = Object.keys(data);
     const style = select('style', styles, styles[0]);
-    const ctaData = object(`Data (${style})`, JSON.stringify(data[style]));
-    console.log('Button', ctaData);
-    //  const data = object(`Data (${stye})`, date[style])
-    // cta[ctaType] = data;
+    const urlBy = {
+      jump: '#example',
+      local: 'https://ibm.com',
+      external: 'https://example.com',
+    };
+
+    switch (style) {
+      case 'text':
+        type = select('type', types, types[0]);
+        data = {
+          type: type,
+          href: urlBy[type],
+          copy: copy[0],
+        };
+        break;
+      case 'card':
+        type = select('type', types, types[0]);
+        data = {
+          type: type,
+          href: urlBy[type],
+          title: copy[0],
+        };
+        break;
+      case 'feature':
+        type = select('type', types, types[0]);
+        data = {
+          heading: copy[0],
+          card: {
+            type: type,
+            href: urlBy[type],
+            title: copy[1],
+            image: {
+              defaultImage: 'https://picsum.photos/id/672/672',
+              alt: '...',
+            },
+          },
+        };
+        break;
+      case 'button':
+        type = [
+          select('button 1 type', types, types[0]),
+          select('button 2 type', types, types[0]),
+        ];
+
+        data = [
+          {
+            type: type[0],
+            href: urlBy[type[0]],
+            copy: copy[0],
+          },
+          {
+            type: type[1],
+            href: urlBy[type[1]],
+            copy: copy[1],
+          },
+        ];
+        break;
+      default:
+    }
 
     return (
-      <CTA
-        // cta={select('ctaType', cta, cta.card)}
-        //type={select('type', type, type.local)}
-        style={style}
-        cta={data.button}
-        // buttons={buttons}
-      />
+      <div className="bx--grid ">
+        <div className="bx--row">
+          <div className="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
```

Comment:
Added the grid to make it a little more digestible within storybook. In particular for the FeatureCard.